.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Please exclude the personal settings file.
Would be great if you add a `.gitignore` file to he repo.
e.g. https://www.toptal.com/developers/gitignore/api/visualstudio,visualstudiocode,csharp,dotnetcore

Reply:
The same applies for the file `.vs/tributech-dsk-api-clients/v16/.suo`.

Reply:
The `.vs/tributech-dsk-api-clients/v16/.suo` is still included in the PR but should also be removed.

Reply:
And please also add the `.gitignore` which is currently not yet on the remote branch.